jax/experimental/maps.py

```diff
@@ -904,6 +904,22 @@ def new_out_axes_thunk():
 batching.SPMDBatchTrace.process_xmap = partialmethod(_batch_trace_process_xmap, True)  # type: ignore
 
 
+def _xmap_initial_to_final_params(params):
+  out_axes_thunk = HashableFunction(lambda: params['out_axes'],
+                                    closure=params['out_axes'])
```

Comment:
I still don't understand where `out_axes` is coming from and why it can't be turned into a thunk there, but I assume there's a good reason :)